net/host.go

```diff
@@ -74,6 +74,11 @@ func NewHost(cfg *p2pnet.Config) (*Host, error) {
 	}, nil
 }
 
+// P2pHost returns the underlying go-p2p-net host.
+func (h *Host) P2pHost() P2pnetHost {
```

Comment:
Does the word "Net" add anything to that interface type name? I was going to suggest a capital `N`, since the source package, `go-p2p-net`, is treating them as separate words, but if the function is called `P2pHost`, maybe `net` isn't adding any value to the type name either.

Reply:
yeah I wasn't sure what to call this either, I felt `P2pNetHost` would be too redundant

Reply:
updated to `P2pHost`